frontend/src/pages/Admin/Dashboard.jsx

```
@@ -1,129 +1,112 @@
-		<div className="stat-title">{title}</div>
-		<div className="stat-value">{value}</div>
-		<div className="stat-desc mt-2">{desc}</div>
-	</div>
+  <div className="bg-red-200 p-4 rounded-lg shadow-md text-center">
+    <div className="flex justify-center mb-2">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 24 24"
+        className="h-8 w-8 stroke-current text-secondary"
+      >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)